operations/commit_queue.go

```diff
@@ -173,27 +182,79 @@ func setModuleCommand() cli.Command {
 	}
 }
 
-func listCommitQueue(ctx context.Context, client client.Communicator, projectID string) error {
+func listCommitQueue(ctx context.Context, client client.Communicator, rc *legacyClient, projectID string, uiServerHost string) error {
 	cq, err := client.GetCommitQueue(ctx, projectID)
 	if err != nil {
 		return err
 	}
+	projectRef, err := rc.GetProjectRef(projectID) // rc versus ac?
+	if err != nil {
+		return errors.Wrapf(err, "can't find project for queue id '%s'", projectID)
+	}
+	grip.Infof("Project: %s\n", projectID)
+	grip.Infof("Type of queue: %s\n", projectRef.CommitQueue.PatchType)
```

Comment:
will this be something the user wants to know?

Reply:
I wonder if it's useful considering the two types have different formats. It's not a requirement though, could be cut.